.clang-tidy

```diff
@@ -19,7 +19,9 @@ Checks: >-
   -modernize-use-default-member-init,
   -modernize-use-emplace,
   -modernize-use-override,
+  -modernize-use-trailing-return-type,
```

Comment:
seems like something went wrong with my rebase :(

Reply:
I wonder if there's a way around this. Yesterday, when I did the interactive rebase, I had to go through > 200 commits (1-by-1) - and there were many merge conflict within every single one. This took be about 6 hours. I've got no clue what caused those conflicts, because there were no commits on master really affecting this area of the code, unless I missed something (which is very possible). 

I'll do some investigation next weekend, but right now, I don't know what to do with this. I'm using SourceTree (git client with UI) rather than command line git - mostly because when I tried to do it via command line it would take even longer to go through it, so I stopped at some point. Either way, hopefully I'll find out what went wrong then ...

Reply:
Found out quite a few issues, I definitely shouldn't use SourceTree the way I did. Might be actually easier to get it done properly via command line after all.